packages/renderer/src/lib/container/ContainerList.svelte

```diff
@@ -372,7 +373,7 @@ $: containersAndGroups = containerGroups.map(group =>
     {#if selectedItemsNumber > 0}
       <div class="inline-flex space-x-2">
         <Button
-          on:click="{() => deleteSelectedContainers()}"
+          on:click="{() => withBulkConfirmation(deleteSelectedContainers, `delete ${selectedItemsNumber} containers`)}"
```

Comment:
Software with incorrect text like '1 containers' is a pet peeve of mine. :) Could you please either use an if statement or a `{ selectedItendsNumber > 1 ? 's':''}` block so that it is '1 container', '2 containers'? (or reword to avoid it somehow)